mlir/test/Integration/Dialect/SparseTensor/CPU/block.mlir

```diff
@@ -10,9 +10,10 @@
 // DEFINE: %{compile} = mlir-opt %s --sparsifier="%{sparsifier_opts}"
 // DEFINE: %{compile_sve} = mlir-opt %s --sparsifier="%{sparsifier_opts_sve}"
 // DEFINE: %{run_libs} = -shared-libs=%mlir_c_runner_utils,%mlir_runner_utils
+// DEFINE: %{run_libs_sve} = -shared-libs=%mlir_native_utils_lib_dir/libmlir_runner_utils%shlibext,%mlir_native_utils_lib_dir/libmlir_c_runner_utils%shlibext
```

Comment:
Why can't this use the substitutions you've defined? Same question for the other parse tests.
```suggestion
// DEFINE: %{run_libs_sve} = -shared-libs=%native_mlir_runner_utilst,%native_mlir_c_runner_utils
```

Reply:
ah, totally missed this part